### DISCUSSION TITLE
Twitter Bot Support?!?! Also another issue.

### DISCUSSION THREAD

**Discussion by ghost** [upptime/upptime#621] [Ideas]
Hi there, wondering if there is a plan on getting Twitter bot support? It would absolutely get more users over!

Right now I’m using email to tweet using zapier.
But for some reason, when the services get back up emails aren’t sent (only if it’s been more than 10 minutes of downtime).
Is there anything I have to change?

**Comment by AnandChowdhary** [comment]
Twitter bot support is a good idea! We delete issues which seem like false positives so maybe that's why the email is not triggered?

**Reply by ghost** [reply]
Probably

**Reply by ghost** [reply]
I think you should add an option to either delete false positives or to have the normal “up” option but mentioning it could have been a false positive.
